bin/check-log.rb

```diff
@@ -128,6 +128,13 @@ class CheckLog < Sensu::Plugin::Check::CLI
          proc: proc(&:to_i),
          default: 250
 
+  option :encode_utf16,
```

Comment:
I see your point that being said its trivial to set the encoding on the regex itself, if you still feel strongly about it I am willing to conceded as long as you add some comments in the code around it so a year from now we understand why it was done that way.